tools/etcd-dump-db/README.md

```diff
@@ -38,7 +38,7 @@ meta
 ```
 
 
-#### hash [data dir or db file path]
+#### hash `[data dir or db file path]`
```

Comment:
It's possible to have links in the header though. I would suggest making the heading the command verb and then listing the usage below (/cc @joshix).

Maybe something like:

#### hash

Computes the hash of db file.

```
Usage:
  etcd-dump-db hash [data dir or db file path] [flags]

Flags:
  -h, --help   help for hash
```

```
$ etcd-dump-db hash agent01/agent.etcd
db path: agent01/agent.etcd/member/snap/db
Hash: 3700260467


$ etcd-dump-db hash agent02/agent.etcd

db path: agent02/agent.etcd/member/snap/db
Hash: 3700260467


$ etcd-dump-db hash agent03/agent.etcd

db path: agent03/agent.etcd/member/snap/db
Hash: 3700260467
```